example/src/main/java/io/netty/example/http2/helloworld/client/Http2Client.java

```diff
@@ -77,8 +76,7 @@ public static void main(String[] args) throws Exception {
                     SelectorFailureBehavior.NO_ADVERTISE,
                     // ACCEPT is currently the only mode supported by both OpenSsl and JDK providers.
                     SelectedListenerFailureBehavior.ACCEPT,
-                    SelectedProtocol.HTTP_2.protocolName(),
-                    SelectedProtocol.HTTP_1_1.protocolName()))
+                    "h2", "http/1.1"))
```

Comment:
Yes. I coulnd't find a good place to put the constants. Suggestions?
We'll have to put spdy/3.1 and maybe some more there as well.